chi2/chinotype.py

```diff
@@ -4,9 +4,9 @@
    by relative prevalence of concepts.
 
 Usage:
-   driver.py [options][-f PATTERN]... -m QMID
-   driver.py [options][-f PATTERN]... -p PSID
-   driver.py [options][-f PATTERN]... -t PSID -r PSID
+   chinotype.py [options][-f PATTERN]... -m QMID
```

Comment:
Assume driver.py is what the old name of the script was, changed to chinotype.py